api/src/services/region-service.ts

```diff
@@ -0,0 +1,23 @@
+import { IDBConnection } from '../database/db';
+import { RegionRepository } from '../repositories/region-repository';
+import { DBService } from './db-service';
+
+export class RegionService extends DBService {
+  regionRepository: RegionRepository;
+
+  constructor(connection: IDBConnection) {
+    super(connection);
+
+    this.regionRepository = new RegionRepository(connection);
+  }
+
+  /**
+   * This function will calculate and save regions (via polygon intersect) for a given submission id.
+   *
```

Comment:
Could you describe in a bit more detail what `intersectThreshold` does?

Reply:
I see you described it in the repo method. Probably would be useful to copy the whole comment over?

Reply:
I copied over the comment from the repo and updated to account for the extra insert call. Do the comments make sense to you?

Reply:
Looks great!